src/components/Logo/Logo.module.scss

```diff
@@ -1,22 +1,29 @@
-.logoName {
-  font-style: normal;
-  font-weight: 400;
-  font-size: 22px;
-  height: 3.5vh;
-}
-
 .logoLabel {
   font-style: normal;
-  font-weight: 100;
-  font-size: 15px;
+  font-weight: 600;
+  font-size: 12px;
   height: auto;
+  color: var(--white-color);
+
+  @media (max-width: 768px) {
+    width: 50vw;
```

Comment:
for text, we only ever want to use max-width. we can allow it to be contained, but shouldn't allow it to take up more size than it needs.